var/spack/repos/builtin/packages/qt/package.py

```diff
@@ -154,6 +156,7 @@ class Qt(Package):
     # Dependencies, then variant- and version-specific dependencies
     depends_on("icu4c")
     depends_on("jpeg")
+    depends_on('gmake')
```

Comment:
It worked for me too but was using the system gmake. I will try to determine where and when this was introduced but it showed up in the configure logs.

Reply:
Weird... is this on a linux system? On macOS I don't have a system gmake but qt 5.14 configures without a problem.

Reply:
Yes, linux system (ubuntu 21.10). I think it may be related to the pkg-config issue: qt build picks up /usr/bin/pkg-config, which probably returns the /usr/bin/gmake instead of the one in spack. I can't find anyplace where gmake is explicitly included in their configure scripts, but it definitely used in the logs when testing for packages, e.g. lines like `cd ${build_stage}/spack-stage-qt-5.15.2-z2yjs6tgrgwfogzjwhcbnfwg3kgkiedp/spack-src/config.tests/verifyspec && MAKEFLAGS= /usr/bin/gmake`. I'm trying to force qt configure to use the spack pkg-config, but not having any luck for now.

Reply:
I think it's fine to remove the explicit gmake dependency. Spack assumes `make` is available universally, and even if qt picks up a different make/gmake, it won't affect the final product since it's a build-only dependency.

Reply:
```suggestion
```